src/main/java/com/massivedisaster/bintraydeployautomator/utils/CommandLineUtils.java

```diff
@@ -24,16 +32,18 @@
             cmd = parser.parse(options, args);
         } catch (ParseException e) {
             System.out.println(e.getMessage());
-            formatter.printHelp("Java -jar BintrayDeployAutomator-0.0.4.jar", options);
+            formatter.printHelp("Java -jar BintrayDeployAutomator-" + com.massivedisaster.bintraydeployautomator.utils.Version.VERSION + ".jar", options);
```

Comment:
`BintrayDeployAutomator-" + com.massivedisaster.bintraydeployautomator.utils.Version.VERSION + ".jar` is used in at least two palces. should be a final variable.

Reply:
Fixed in the next commit.